<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityTagsForm_template</class>
 <widget class="QDialog" name="ActivityTagsForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the activity tags dialog">Activity tags</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="activityTagsListWidget"/>
     <widget class="QPlainTextEdit" name="currentActivityTagTextEdit"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addActivityTagPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renameActivityTagPushButton">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeActivityTagPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commentsPushButton">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="printablePushButton">
       <property name="text">
        <string>Printable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="notPrintablePushButton">
       <property name="text">
        <string>Not printable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="activateActivityTagPushButton">
     <property name="text">
      <string>Activate all activities for selected tag</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="deactivateActivityTagPushButton">
     <property name="text">
      <string>Deactivate all activities for selected tag</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="moveActivityTagUpPushButton">
       <property name="text">
        <string comment="It means move the item up in the list">Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveActivityTagDownPushButton">
       <property name="text">
        <string comment="It means move the item down in the list">Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortActivityTagsPushButton">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
